aerolyzer/image_restriction_main.py

```diff
@@ -0,0 +1,53 @@
+'''
+Aerolyzer
+March 2, 2017
+Image Restriction Main File
+Description: This file is called by the website when a photo is uploaded, and uses
+information from image_restriction_functions and image_restrictions_conf to verify that
+all image restrictions have been met.
+'''
+
+'''
+import libraries
+'''
+import sys
+
+'''
+import references
+'''
+from image_restriction_conf import imageRestrictionCriteria as Criteria
+from image_restriction_functions import imageRestrictionFunctions as Functions
+    
+    
+'''
+Purpose:        The purpose of this main function is to check all image restrictions and
+                produce the correct error message should one occur.
+Inputs:         string image (as sys.argv[1])
+Outputs:        None
+Returns:        N/A
+Assumptions:    N/A
+'''
+def main():
+    #instantiate classes
+    functions = Functions()
+    criteria = Criteria()
+
+    #Retrieve exif data
+    exifData = functions.get_exif(sys.argv[1])
```

Comment:
Update reading in this parameter from command line such that if none is provided an error msg/ usage msg is given to the user. 
Better yet, provide an image (in the image folder). If no parameter is given, in addition to the error msg, you can run by default using that image.